builder/dockerfile/dispatchers.go

```diff
@@ -414,6 +414,8 @@ func cmd(b *Builder, args []string, attributes map[string]bool, original string)
 	}
 
 	b.runConfig.Cmd = strslice.StrSlice(cmdSlice)
+	// set config as already being escaped, this prevents double escaping on windows
+	b.runConfig.ArgsEscaped = true
```

Comment:
Let me look into that. Gut reaction is no, but if it turns out the answer is yes, I can follow on with a separate PR.